internal/requests/requests.go

```diff
@@ -7,6 +7,17 @@ import (
 	"github.com/argoproj-labs/argo-cloudops/internal/validations"
 )
 
+type TxIDHeaderType string
```

Comment:
wondering if this should be more generic. maybe `ContextKey` or `HeaderKey` or similar?

Reply:
i'll change it to HeaderKey